Bottom: 792bc2f356653d9189164dfd95f291208b94147b
Top:    6d6d9b2a7a94b6ead3285f418f7495b7cc33d2f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:38:01 +0100

Refresh of hid-gtk-rework-zoom-pan-api

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 3f9b51a..4cd4d22 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -32,6 +32,8 @@ RCSID ("$Id$");
 
 bool ghid_flip_x = false, ghid_flip_y = false;
 
+static void ghid_zoom_view_fit (void);
+
 static void
 ghid_pan_view_abs (Coord pcb_x, Coord pcb_y, int widget_x, int widget_y)
 {
